package.json

```diff
@@ -44,6 +44,7 @@
     "release": "./script/upload.py -p",
     "repl": "python ./script/start.py --interactive",
     "start": "python ./script/start.py",
-    "test": "python ./script/test.py"
+    "test": "python ./script/test.py",
+    "test-asar-integrity": "./spec/node_modules/.bin/mocha ./spec/asar-integrity-test"
```

Comment:
You can run a new instance of Electron as part of test:

```js
// A normal test
it('test', function() {
  execFileSync(remote.process.execPath, ['fixtures/sometest.js' ])
})
```

```js
// fixtures/sometest.js
app.on('ready', () => {
  if (error)
    app.exit(1)
  else
    app.exit()
})
```

The `app.relaunch` test in api-app-spec.js uses this method to test an extra app.